Philips.CodeAnalysis.MaintainabilityAnalyzers/Maintainability/NoHardCodedPathsAnalyzer.cs

```diff
@@ -45,14 +46,20 @@ private void Analyze(SyntaxNodeAnalysisContext context)
 				return;
 			}
 
-			//if the character of the string do not match either of the characters : for windows and / for linux; no need to run regex, simply return.
+			// If the character of the string do not match either of the characters : for windows and / for linux; no need to run regex, simply return.
```

Comment:
The second clause of line 50 appears to be a Linux remnant. Suggest replacing it with !pathValue[1].Equals(':') || !pathValue[2].Equals('\\')

Reply:
Updated for windows